migration/ocpmigrations/ocp_migrations_test.go

```diff
@@ -0,0 +1,29 @@
+package ocpmigrations_test
+
+import (
+	"database/sql"
+	"testing"
+
+	"github.com/RedHatInsights/insights-results-aggregator/migration"
+	"github.com/RedHatInsights/insights-results-aggregator/storage"
+	ira_helpers "github.com/RedHatInsights/insights-results-aggregator/tests/helpers"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+func prepareDB(t *testing.T) (*sql.DB, types.DBDriver, func()) {
+	mockStorage, closer := ira_helpers.MustGetMockStorage(t, false)
+	dbStorage := mockStorage.(*storage.OCPRecommendationsDBStorage)
+
+	return dbStorage.GetConnection(), dbStorage.GetDBDriverType(), closer
+}
+
+func prepareDBAndInfo(t *testing.T) (*sql.DB, types.DBDriver, func()) {
+	db, dbDriver, closer := prepareDB(t)
+
+	if err := migration.InitInfoTable(db); err != nil {
+		closer()
```

Comment:
I think this should always be called, since the connection is established at some point to create the test DB

Reply:
@epapbak we don't want to close it after `prepareDBAndInfo` finishes, it returns the closer function which is then used/deferred in all the functions that use this prepare function

(this code and the `actual_migrations_test.go` is a 1:1 copy from the code that was always here)